transcribed/page0010.txt

```diff
@@ -1,56 +1,56 @@
-001             TITLE ITS 1.0 3/19/67                                           PAGE 1
-002             MOBY==1
```

Comment:
I don't see a space after the == operator.

Reply:
neither do I!